strr-platform-web/app/components/connect/form/phoneNumber/CountryCode.vue

```diff
@@ -0,0 +1,121 @@
+<script setup lang="ts">
+// NOTE: country-codes-list fails if not overridden (even though it is in the repo base layer)
```

Comment:
FYI - this was weird and I think it's due to a combination of the moduel 'country-codes-list' and the way Nuxt imports from github repo extensions (it is different than when it extends from npm packages). Will experiment more with this later, but for now we need a copy of ConnectFormPhoneNumberCountryCode in each layer. I think this will go away once its coming from an npm package vs git repo